src/redis-cli.c

```diff
@@ -46,8 +46,8 @@
 #include <limits.h>
 #include <math.h>
 
-#include "hiredis.h"
-#include "sds.h"
+#include <hiredis.h>
+#include <sds.h> /* use sds.h from hiredis, so that only one set of sds functions will be present in the binary */
```

Comment:
Yep your reasoning makes sense to me, I'm wondering if we should switch all the implementations inside Redis to a single one. But I don't remember if the current hiredis implementation was a bit different. For sure if it was different, the difference was only marginal. Maybe as part of this pull request, I'll try to unify at least sds of Redis, Disque, and antirez/sds :-) Cheers.